README.md

```diff
@@ -35,9 +35,20 @@
 
 ## Releasing OpenSearch
 
+### About OpenSearch releases and versions
+
+OpenSearch and OpenSearch Dashboards are distributed as bundles that include both core engines and plugins. OpenSearch [follows semantic versioning](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/). Software, such as Data Prepper, clients, and the Logstash output plugin, are versioned independently of the OpenSearch and OpenSearch Dashboards. Because of this, they may have independent releases from the main OpenSearch and OpenSearch Dashboards distribution. In addition to the semantic versioning , OpenSearch may release software under alpha, beta, release candidate, and generally available labels. Below is a definition of when to use each label.
+
+Release labels:
+
+* **Alpha** - The code is released with instructions to build. Built distributions of the software may not be available. Some features many not be complete. Additional testing and developement work is planned. Distributions will be prefixed with `alphaX` where "X" is the number of the alpha version.
```

Comment:
@peterzhu1992 and @dblock would be of your interest.

Reply:
@dblock and @peterzhu1992 is the labeling @stockholmux is mentioning feasible? e.g., `1.0.0-alpha.2`

Reply:
Are there many products using this? I mostly see `rc1` and `alpha1` out there, and that would be a change from what Elasticsearch (see https://github.com/opensearch-project/OpenSearch/blob/8fbd8cd998637029a6d63a7f401af562cb56c3c6/buildSrc/src/test/java/org/opensearch/gradle/BwcVersionsTests.java#L63 for version history). It's technically feasible I am sure with more work. Open an issue?

Reply:
I believe @stockholmux pulled this directly from [item 9](https://semver.org/#spec-item-9) in the semver spec. Like the below. 

_Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7, 1.0.0-x.7.z.92, 1.0.0-x-y-z.–._

Either approaches would work for me. The requirement is numbering pre-release software incase there is another iteration of it we want to release. If projects in practice use -rc1, alpha1, etc. over -rc.1, alpha.1, I'm okay with using -rc1, etc.

Reply:
Actually it was [item 11](https://semver.org/#spec-item-11) as well. If you digest the SemVer spec, it has a some really good reasons for using separate identifiers. 11.4.2 says "Identifiers with letters or hyphens are compared lexically in ASCII sort order." 

So, let's think for a moment - if we _ever_ get into a situation where we have a double digit alpha release the order will be messed up. It would be: `1.0.0-alpha1` > `1.0.0-alpha10` > `1.0.0-alpha2` etc.

Reply:
That's a good point, but let's not have 10 RCs :) Also note that OpenSearch has a hugely complex internal mechanism to compare versions today, and it's nothing like "assume semver to do the comparison".

I vote to leave `rc1` alone.

Reply:
@dblock So, one thing to keep in mind that this isn't really _for_ OpenSearch, it's for the users and integrations. 

Whilst OpenSearch has the mechanism to make the right version ordering, assuming that other integrating software has this understanding is probably a bad assumption. Saying we use SemVer then not adopting this part of the spec is potentially creating pain for our users and technical debt down the line.

Reply:
True, but only if we have more than 9 builds with the same qualifier, right? I am not saying you're wrong, but the actual impact is low while the amount of work may be significant.